### PR TITLE
AARD-1735: Logging module updates

### PR DESCRIPTION
**Pull request by BrandonPacewic**
- Added logging file history:
  - The past 10 log files will now be saved.
- Removed old analytics and timing system.
- Improved / added timing and performance based logging:
  - Example: Added `@timed` decorator to functions added in #981 to track performance concerns.
  - Also added timing to the overall export to show end to end runtime.
- Created a wrapper to avoid global `try` and `except`s that were all leading into `logger.error` and `ui.messageBox` error messages.

> [!important]
> - ~~Blocked by #995.~~

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1735)

### CODE REVIEW

File: exporter/SynthesisFusionAddin/Synthesis.py [hunk 1]
```diff
@@ -28,109 +27,95 @@
...
-    #     A_EP.send_view("open")
-
     # toolbar = Toolbar('SketchFab')
     work_panel = Toolbar.getNewPanel(f"{APP_NAME}", f"{INTERNAL_ID}", "ToolsTab")
 
     commandButton = HUI.HButton(
         "Synthesis Exporter",
         work_panel,
-        Helper.check_solid_open,
+        lambda *_: True,  # TODO: Should be redone with various refactors.
```
Comment (azaleacolburn) [inline exporter/SynthesisFusionAddin/Synthesis.py:118]: 👀 Should this be redone with a refactor now?

**Review by azaleacolburn — APPROVED** [review]
One small question about a comment, otherwise, LGTM

File: exporter/SynthesisFusionAddin/Synthesis.py (hunk 1 above)
Comment (BrandonPacewic) [reply exporter/SynthesisFusionAddin/Synthesis.py:118]: Leaving this for the `HButton` refactor. Doing my best to not change too many things that aren't directly related to the main goal of this PR.

**Review by PepperLola — APPROVED** [review]
Looks good. It created new log files each time and correctly logged debug information. Timing looks like it works. I got an error setting the export destination to upload and it sent a message box and logged it.

**Pull request merged**